### ISSUE TITLE
Shadow breaking change in underlying shiki dependency

### ISSUE THREAD

**Issue by benjag** [nuxt/content#2197]
### Environment

- Operating System: `Darwin`
- Node Version:     `v16.16.0`
- Nuxt Version:     `3.6.5`
- Nitro Version:    `2.5.2`
- Package Manager:  `npm@8.11.0`
- Builder:          `vite`
- User Config:      `app`, `modules`, `css`, `content`
- Runtime Modules:  `@nuxt/content@2.7.2`, `@vueuse/nuxt@9.13.0`
- Build Modules:    `-`

### Reproduction

https://stackblitz.com/edit/github-d56zne?file=content%2Findex.md

### Describe the bug

There was a shadow breaking change in the underlying shiki highlighter dependency when using a `material-*` theme. Shiki renamed the themes causing any existing material theme references in the nuxt config to break and crash the app at runtime.

https://github.com/shikijs/shiki/releases/tag/v0.13.0

I did not see anything in the `nuxt/content` release notes or any other open issues so I figured you may want to call this out in your docs. I had to hunt for a bit to figure out what happened.

**Comment by nobkd** [comment]
You can make a PR to add some info to the docs.

I don't really think that it is needed, because:
- The [highlight option docs](https://content.nuxtjs.org/api/configuration#highlight-options) link to the [Shiki themes list](https://github.com/shikijs/shiki/blob/main/docs/themes.md#all-themes) which states the up-to-date names.
- It has been quite some time (6 month) since the renaming happened in Shiki.
- The error message is relatively clear, as it states, that the theme asset is unknown / not found.

But that's just *my* thoughts.

**Comment by benjag** [comment]
Fair point, but in this case. It appears that the shiki update was brought in to `@nuxt/content` [quite recently](https://github.com/nuxt/content/pull/2125) and that a [test had to updated](https://github.com/nuxt/content/pull/2125/commits/6cca8b5f146ee361d1143ec2894c103a1bb34baa#diff-c0c3a34f47f0c960ab524bb765692005eded7412aec07f961162b3a2f9fba094) to address the breaking change.

Just feel like it would have been worth a mention in the release notes as well.

**Comment by nobkd** [comment]
Good that you mention this, I didn't see that dependency and test update. 
That probably also explains, why #1936 didn't work back then.

Since this was only recently updated here in Nuxt Content, I agree with you, that it's probably good to mention it in the docs.

**Comment by benjag** [comment]
I do think you're correct that it doesn't need to be in the actual documentation, but a quick callout in the changelog would be helpful

**Comment by farnabaz** [comment]
Sorry for causing inconvenient
I will add a note in the release notes, hope others will notice this change before wasting too much time

**Comment by farnabaz** [comment]
Just add a note in both v2.7.1 and v2.7.2 release notes.
Thanks for the issue and raising concern.
